cmd/bff/main.go

```diff
@@ -216,7 +216,7 @@ func announce(c *cli.Context) (err error) {
 	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 	defer cancel()
 
-	if err = client.Login(ctx); err != nil {
+	if err = client.Login(ctx, &api.LoginParams{}); err != nil {
```

Comment:
Is it ok to pass in `nil` instead of an empty struct?

Reply:
Yes it does work.